examples/FasterTransformer_HuggingFace_Bert/README.md

```diff
@@ -0,0 +1,148 @@
+## Faster Transformer 
```

Comment:
Please add a note this works only for GPUs

Reply:
Will be better to move into its own Readme for Faster Transformer and include a reference in the main readme

Reply:
> Please add a note this works only for GPUs

This has been highlighted in the doc, that it requires tensor core gpus.